laws/shared/src/main/scala/algebra/laws/RunSerializationLaws.scala

```diff
@@ -0,0 +1,9 @@
+package algebra.laws
+
+sealed trait SerializationCheck
```

Comment:
With the type parameter, in the companion object of `M` you could put whether to check the serialize check or not. Might be a good idea.

But if we call it `SerializationCheck` I think I'd rather call it `IsSerializable[M]` as a typeclass that is the actual promise that `M` is java serializable and it will always work. Then you could make combinators for containers which are basically the `and` on all of these.

I don't know if we need that level of geekout though. Basically the use case we care about are things like Futures mostly.